tests/Resources/ti.ui.shortcut.addontest.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2020 by Axway, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+
+const should = require('./utilities/assertions');
+
+describe('Titanium.UI.Shortcut', () => {
+
+	it.android('Ti.UI.Shortcut', () => {
+		should(Ti.UI.Shortcut).not.be.undefined();
+	});
+
+	it.android('createShortcut', () => {
+		should(Ti.UI.createShortcutItem).not.be.undefined();
+		should(Ti.UI.createShortcut).be.a.Function();
+
+		// ONLY compatible with Android 7.1+, end test early.
+		if (Ti.Platform.osname === 'android') {
+			const version = Ti.Platform.version.split('.');
+			if (parseInt(`${version[0]}${version[1]}`) < 71) {
+				return;
+			}
+		}
+
+		// Create shortcut.
```

Comment:
I think you should break out separate tests/sub0suites for each property/method here. Right now you're just checking existence and that's fine and it doesn't look like too much, but eventually we'll ant to expand the tests for each to verify existence, default values, return types, basic workflow, etc.